features/bookmarks/bookmark_filter.feature

```diff
@@ -0,0 +1,20 @@
+@bookmarks
+Feature: Filter bookmarks
+  In order to search an archive full of bookmarks
+  As a humble user
+  I want to filter some bookmarks
+
+  @new-search
+  Scenario: Filter a user's bookmarks by work language
+    Given I have loaded the fixtures
```

Comment:
This might be slightly faster and simpler if you used `I have bookmarks of works in various languages to search` here instead. It should be possible to modify the `I have bookmarks of works in various languages to search` step definition to assign the bookmarks to a specific user.

Reply:
as far as connection between feature and steps, thoughts between
```gherkin
# secret knowledge that Given step uses specific user
Given I have bookmarks of works in various languages to search
And I am logged in as "recengine"
```
vs
```gherkin
# with user name param "(.*?)"
Given "recengine" has bookmarks of works in various languages to search
And I am logged in as "recengine"
```
or maybe something else entirely?

Reply:
The version wit the user name param sounds good! Doing things that way is more transparent and more reusable -- I know I have frequently been tripped up or inconvenienced by some of the collection steps that secretly use `mod1` as the user.